app/poros/gym.rb

```diff
@@ -1,10 +1,23 @@
 class Gym
-  attr_reader :yelp_gym_id, :name, :address, :phone
+  attr_reader :yelp_gym_id, :name, :full_address, :address1, :address2, :address3,
+              :city_state_zip, :phone
 
   def initialize(data)
-    @yelp_gym_id = data[:id]
-    @name        = data[:attributes][:name]
-    @address     = data[:attributes][:address]
-    @phone       = data[:attributes][:phone]
+    @yelp_gym_id    = data[:id]
+    @name           = data[:attributes][:name]
+    @full_address   = data[:attributes][:address]
+    @address1       = data[:attributes][:address_details][:address1]
+    @address2       = data[:attributes][:address_details][:address2]
+    @address3       = data[:attributes][:address_details][:address3]
+    @city_state_zip = format_city_state_zip(data)
+    @phone          = data[:attributes][:phone]
+  end
+
+  def format_city_state_zip(data)
+    city = data[:attributes][:address_details][:city]
+    state_abbr = data[:attributes][:address_details][:state]
+    zip_code = data[:attributes][:address_details][:zip_code]
+
+    "#{city}, #{state_abbr} #{zip_code}"
```

Comment:
Thank you for this as well - I think formatting on the FE makes the most sense for flexibility, if we decide we need to change approach in the final hours.  As I evaluated how to format/send from the BE yesterday, it seemed like it might've pigeon-holed if we chopped out / hard-coded the format in the BE PORO, so the thought process was to send the full address details hash straight from Yelp (minus duplicate or unnecessary key `cross_streets`), so we can just manipulate how we need it in the view based on FE display requirements